### PR TITLE
Bug 249

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: volontulo/templates/auth/password_reset.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{% block title %}Resetowanie hasła{% endblock %}
+
+{% block content %}
+    <h2>Resetowanie hasła</h2>
+
+    <form id="user_register" method="post" action="{% url 'password_reset' %}" role="form" enctype="multipart/form-data">
+        {% csrf_token %}
+        <div class="form-group">
+            {{ user_form.email.label_tag }}
+            <input type="text" value="" name="email" class="form-control" id="{{ user_form.email.id_for_label }}" />
```
Comment (filipgorczynski) [inline volontulo/templates/auth/password_reset.html:12]: Here should be something like `value="{{ request.POST.email|default:'' }}"`

File: volontulo/views/auth.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+def login(request):
+    u"""Login view."""
+    if request.method == 'POST':
+        username = request.POST.get('email')
+        password = request.POST.get('password')
+        user = auth.authenticate(username=username, password=password)
+        if user is not None:
+            if user.is_active:
+                auth.login(request, user)
+                messages.add_message(
```
Comment (filipgorczynski) [inline volontulo/views/auth.py]: yield_messages_success/yield_messages_error from volontulo.utils

File: volontulo/views/auth.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        template_name='auth/password_reset.html',
+        post_reset_redirect='login',
+        from_email=FROM_ADDRESS,
+        subject_template_name='emails/password_reset.subject',
+        email_template_name='emails/password_reset.txt',
+        html_email_template_name='emails/password_reset.html',
+    )
+
+
+# pylint: disable=unused-argument
```
Comment (filipgorczynski) [inline volontulo/views/auth.py]: Remove unused pylint comment.

Comment (magul) [inline volontulo/views/auth.py]: done

File: volontulo/templates/auth/password_reset.html (hunk 1 above)
Comment (magul) [inline volontulo/templates/auth/password_reset.html:12]: Should it work like this already? It there will not be request.POST.email template engine will substitute here empty string.

**Pull request merged**